edi_oca/models/edi_backend.py

```diff
@@ -428,6 +422,15 @@ def _check_output_exchange_sync(
                 # TODO: run in job as well?
                 self._exchange_output_check_state(rec)
 
+    def _handle_new_output_exchange_records(self, new_records, skip_send=False):
+        for rec in new_records:
```

Comment:
We should still add some sort of validation here. I have to think about it.
For now, you can keep it as it is for your use case but we'll have to adapt it.